vlog4j-parser/src/test/java/org/semanticweb/vlog4j/syntax/parser/RuleParserTest.java

```diff
@@ -411,4 +412,20 @@ public void testBlankPredicateName() throws ParsingException {
 		RuleParser.parse(input);
 	}
 
+	@Test
+	public void predicateIriRoundTripTest() throws ParsingException {
+		AbstractConstantImpl a = new AbstractConstantImpl("a");
```

Comment:
this is not a roundtrip test.
What do you want to test here?

Reply:
wanted to make sure its the same if i already have < > or not

Reply:
but I guess this is already checked i will remove it

Reply:
please check if this is indeed already tested, and if not, rename the unit test and leave it.

Reply:
done